tests/lib/clients/03-publish-b2c-qos1.py

```diff
@@ -10,7 +10,7 @@ def on_message(mqttc, obj, msg):
     assert msg.topic == "pub/qos1/receive", f"Invalid topic: ({msg.topic})"
     assert msg.payload == expected_payload, f"Invalid payload: ({msg.payload})"
     assert msg.qos == 1, f"Invalid qos: ({msg.qos})"
-    assert msg.retain is not False, f"Invalid retain: ({msg.retain})"
+    assert not msg.retain, f"Invalid retain: ({msg.retain})"
```

Comment:
Isn't this the inverse test c.f. before?

Reply:
Yes, but I think the test was wrong.
* We don't produce a message with retain flag in this test, so it make sense that msg.retain is False
* Previously, msg.retain was an interger so `msg.retain is not False` was always true.
* I've checked and the value was `0`. So it seems expected to assert the value `False`

Reply:
Maybe this change should be done in a separate PR, then? I'm afraid it will confuse future code archeologists terribly if this happens in a commit that says "Add typing".